src/jit/alloc.cpp

```diff
@@ -81,11 +83,8 @@ size_t ArenaAllocator::getDefaultPageSize()
 // ArenaAllocator::ArenaAllocator:
 //    Default-constructs an arena allocator.
 ArenaAllocator::ArenaAllocator()
-    : m_memoryManager(nullptr)
-    , m_firstPage(nullptr)
-    , m_lastPage(nullptr)
-    , m_nextFreeByte(nullptr)
-    , m_lastFreeByte(nullptr)
+    : m_memoryManager(nullptr), m_firstPage(nullptr), m_lastPage(nullptr), m_nextFreeByte(nullptr),
```

Comment:
This looks like a step backwards to me, it's easier to compare the initialization list with data members definitions to see if anything is missing if there's one initialization per line.